extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -395,13 +395,23 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		}
 	}
 
+	private get useVSCodeNotebooks(): boolean {
+		let workbench = vscode.workspace.getConfiguration('workbench');
```

Comment:
Do we know how expensive these calls are to make? Have you considered caching the result and listening on a configuration change instead?

Reply:
They're quite cheap - VS Code caches configuration on the ext host side so it's mostly just a direct lookup in a map 

https://github.com/Microsoft/azuredatastudio/blob/8a3d08f0dec1cdf47d7fc442f90fb2227b828375/src/vs/workbench/api/common/extHostConfiguration.ts#L157

We haven't seen any problems with doing lookups so far so I don't see any issue with doing that here since it's simpler, but caching it is fine too if you'd prefer.

Reply:
ok, that's fine then. let's keep this as is.